Riot/Views/RoomList/PublicRoomTableViewCell.xib

```diff
@@ -1,11 +1,11 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.XIB" version="3.0" toolsVersion="11762" systemVersion="15G1108" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" colorMatched="YES">
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note: During my tests, I observed the dynamic text size was very well supported in this cell.
We should use this xib file as reference to adjust the other cells in the coming PRs.